interactive/api/static/examples/CHAPMAN/camp_data/config.json

```diff
@@ -1,7 +1,7 @@
 {
   "camp-files" : [
-    "camp_data/tolerance.json",
-    "camp_data/species.json",
-    "camp_data/reactions.json"
+    "species.json",
+	"reactions.json",
+    "tolerance.json"
```

Comment:
A minor thing, but can we use the uniform indentation?

Reply:
Changed to tabs in Notepad++.